monitorix.cgi

```diff
@@ -448,6 +436,11 @@ if($RRDs::VERSION > 1.2) {
 	if($RRDs::VERSION >= 1.3) {
 		push(@version12_small, "--font=DEFAULT:0:Mono");
 	}
+
+	push(@version12, "--alt-autoscale", "--allow-shrink" );
+	push(@version12_small, "--alt-autoscale", "--allow-shrink" );
+
+
 }
```

Comment:
Why do you need to include these lines?

Reply:
just a proposal, charts looks better

Reply:
No, they are shifted to the top and there is a useless gap in the bottom.
Monitorix already has a specific option to include extra RRDtool arguments.

This change is not acceptable.